mountpoint-s3/src/cli.rs

```diff
@@ -734,6 +734,9 @@ pub fn create_s3_client(args: &CliArgs) -> anyhow::Result<(S3CrtClient, EventLoo
     if let Some(interfaces) = &args.bind {
         user_agent.key_value("mp-nw-interfaces", &interfaces.len().to_string());
     }
+    if args.cache_express_bucket_name().is_some() {
```

Comment:
Have we considered adding a value to "mp-cache" instead?

Reply:
Hm, do you mind adding an example of the proposed resulting header? Right now it looks like this for multilevel cache:
```
"user-agent": "mountpoint-s3/1.10.0-unofficial+7d01885-dirty mountpoint-s3-client/0.11.0-7d01885-dirty os/linux#<removed> md/arch#x86_64 md/mp-cache md/mp-cache-ttl#indefinite md/mp-cache-express CRTS3NativeClient/0.1.x platform/unknown"
```

Note the `md/mp-cache` and `md/mp-cache-express` values.

Reply:
Are you thinking `md/mp-cache#express` / `md/mp-cache#disk` / `md/mp-cache#multilevel`? May it break any existing logic?

Reply:
Discussed a little offline, agreed to delimit the caches in use by `+`.

For example:
- `md/mp-cache#disk`
- `md/mp-cache#express`
- `md/mp-cache#disk+express`

Reply:
updated to use the following values:
- `md/mp-cache#shared`
- `md/mp-cache#local`
- `md/mp-cache#shared+local`